docs/methods/user.update.md

```diff
@@ -0,0 +1,94 @@
+# user.update
+
+`POST /api/v1/user.update`
+
+## Description
+
+Updated user with the specified properties.
+
```

Comment:
Worth adding a note to explain this API method accepts partial updates, e.g.

Please note: the `user.update()` API method accepts partial updates, meaning you need only specify the properties you want to change instead of supplying full user properties every single time.